packages/ui/src/redux/slices/settings.ts

```diff
@@ -0,0 +1,29 @@
+import { createSlice, Draft, PayloadAction } from "@reduxjs/toolkit";
+
+export enum ThemeMode {
```

Comment:
As this might be used in other places as well, let's move it to file `src/types.ts` where we put all the shared types.